src/Knp/Menu/Twig/Helper.php

```diff
@@ -142,4 +165,30 @@ private function castMenu($menu)
 
         return $menu;
     }
+
+    /**
+     * @param ItemInterface $item
+     *
+     * @return ItemInterface|null
+     */
+    private function retrieveCurrentItem(ItemInterface $item)
+    {
+        $currentItem = null;
+
+        if ($this->matcher->isCurrent($item)) {
```

Comment:
How do we handle the case where there are multiple matching items? i.e. the first item may not be deepest.

```
Pages/ # route: page_list
    List  # route: page_list   <-- want this one   
    Edit # route: page_edit
```